README.md

```diff
@@ -90,6 +90,9 @@ There are 4 main components/concepts in appdash:
 
 Appdash has clients available for Go, Python (see `python/` subdir) and Ruby (see https://github.com/bsm/appdash-rb).
 
+## OpenTracing Support
+
+Appdash supports the [OpenTracing](http://opentracing.io) API. Please see the `/opentracing` subdir for the Go implementation.
```

Comment:
Replace `/opentracing` with just `opentracing`.

Add `... for the Go implementation, or see [GoDoc]([the GoDocs](https://godoc.org/sourcegraph.com/sourcegraph/appdash/opentracing) for API documentation.` and remove `opentracing/README.md` for now since it's so small.